iceberg/iceberg-catalog/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -180,7 +181,8 @@ protected void doRefresh() {
       throw new RuntimeException("Interrupted during refresh", e);
     }
 
-    refreshFromMetadataLocation(metadataLocation);
+    refreshFromMetadataLocation(metadataLocation, HiveConf.getIntVar(conf,
```

Comment:
Is it possible that Spark users would fail here? I think they might not necessarily use a HiveConf and would therefore not have access to our default value of 2, so they'd probably fail here